test/commands/data/dataBulk.nut.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as path from 'path';
+import { strict as assert } from 'node:assert/strict';
+import * as fs from 'fs';
+import * as os from 'os';
+import { expect } from 'chai';
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { sleep } from '@salesforce/kit';
+import { ensurePlainObject } from '@salesforce/ts-types';
+import { BulkResultV2 } from '../../../src/types';
+import { QueryResult } from './dataSoqlQuery.nut';
+
+let testSession: TestSession;
+
+/** Verify that the operation completed successfully and results are available before attempting to do stuff with the results */
+const isCompleted = async (cmd: string): Promise<void> => {
+  let complete = false;
+  while (!complete) {
+    // eslint-disable-next-line no-await-in-loop
+    await sleep(2000);
+    const result = execCmd<BulkResultV2>(cmd);
+    // eslint-disable-next-line no-console
+    if (result.jsonOutput?.status === 0) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      if (result.jsonOutput.result.jobInfo.state === 'JobComplete') {
+        complete = true;
+      } else {
+        // eslint-disable-next-line no-console
+        console.log(result);
+      }
+    } else {
+      // eslint-disable-next-line no-console
+      console.log(result);
```

Comment:
Is this debugging?  logging to the console during tests can really clutter test output.

Reply:
Removed